runtime/doc/lua.txt

```diff
@@ -1581,4 +1581,70 @@ uri_to_fname({uri})                                       *vim.uri_to_fname()*
                 Return: ~
                     Filename
 
+==============================================================================
+Lua module: diff                                                    *lua-diff*
```

Comment:
not a blocker for this PR, but FYI this may be overwritten by the docs generator.

Reply:
can be checked by running the vimdoc generator locally